legacy/documentation/rap/LC_MESSAGES/django.po

```diff
@@ -1976,7 +1977,7 @@ msgstr ""
 #: templates/help/studio/tips/howto/wedo2win-intro.html:30 tips_config.py:257
 #: tips_config.py:267
 msgid "Troubleshooting"
-msgstr ""
+msgstr "Resuelve tus problemas"
```

Comment:
Just to make sure I'm following, is the idea that Rapa Nui has a combination of native words and Spanish words, from being a part of Chile?

Reply:
Yes, they're using Spanish if there isn't a Rapa Nui translation